ui/package.json

```diff
@@ -19,6 +20,7 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
+    "@redux-devtools/extension": "3.3.0",
```

Comment:
Simpler way to set up Redux devtools: https://github.com/reduxjs/redux-devtools/tree/main/extension#13-use-redux-devtoolsextension-package-from-npm